data/index.html

```diff
@@ -15,35 +15,35 @@
             .link6:hover { color: white; background-color: #a00000;}
-            .link7 { padding-bottom: 4px; padding-top: 4px; text-decoration: none; color: #033cd2;}
+            .link7 { padding:8px; text-decoration: none; color: #033cd2; background-color: rgba(211, 211, 211, 0.2); border-radius: 10px; transition: background-color 0.5s;}
             .link7:hover { color: white; background-color: #033cd2;}
-            .link8 { padding-bottom: 4px; padding-top: 4px; text-decoration: none; color: #6e008c;}
+            .link8 { padding:8px; text-decoration: none; color: #6e008c; background-color: rgba(211, 211, 211, 0.2); border-radius: 10px; transition: background-color 0.5s;}
             .link8:hover { color: white; background-color: #6e008c;}
-            .link9 { padding-bottom: 4px; padding-top: 4px; text-decoration: none; color: #6e008c;}
+            .link9 { padding:8px; text-decoration: none; color: #6e008c; background-color: rgba(211, 211, 211, 0.2); border-radius: 10px; transition: background-color 0.5s;}
             .link9:hover { color: white; background-color: #6e008c;}
-            .link10 { padding-bottom: 4px; padding-top: 4px; text-decoration: none; color: #033cd2;}
+            .link10 { padding:8px; text-decoration: none; color: #033cd2; background-color: rgba(211, 211, 211, 0.2); border-radius: 10px; transition: background-color 0.5s;}
             .link10:hover { color: white; background-color: #033cd2;}
-            .link11 { padding-bottom: 4px; padding-top: 4px; text-decoration: none; color: #006e00;}
+            .link11 { padding:8px; text-decoration: none; color: #006e00; background-color: rgba(211, 211, 211, 0.2); border-radius: 10px; transition: background-color 0.5s;}
             .link11:hover { color: white; background-color: #006e00; }
-            .link12 { padding-bottom: 4px; padding-top: 4px; text-decoration: none; color: #033cd2;}
+            .link12 { padding:8px; text-decoration: none; color: #033cd2; background-color: rgba(211, 211, 211, 0.2); border-radius: 10px; transition: background-color 0.5s;}
             .link12:hover { color: white; background-color: #033cd2;}
             .center {text-align: center; position: static; border: 0}
-            .search{background: #FFFFFF; border-radius: 25px; border-width: 1px; border-color: #686868; height: 32px; width: 45%; outline: 0; padding: 0; padding-left: 5px; padding-right: 5px; margin: 0}
-            .btn{background: #5A89EB; border-radius: 25px; width: 15%; height: 32px; color:#FFFFFF; font-style: normal; border-color: #FFFFFF; padding:0; margin: 0; margin-left: 5px; outline: 0; cursor: pointer}
+            .search{background: #0000; border-radius: 25px; border-width: 1px; border-color: #686868; height: 32px; width: 45%; outline: 0; padding: 0; padding-left: 5px; padding-right: 5px; margin: 0;}
```

Comment:
Style is to use six digit hexadecimal colours or names.  You introduce a four digit colour.  Should you use `black`?

Reply:
To be honest, I thought that using a four-digit code for black wouldn't be an issue. I chose black because even when I tried other colors like white or blue (e.g., 4285F4), the issue persisted. As I mentioned in my pr message, this "solution" I provided for the search-box is more of a temporary workaround, rather than a perfect fix, since I don't understand the reason behind the missing corners.